src/guide/agent-functions/defining-agent-functions.rst

```diff
@@ -87,6 +87,26 @@ Runtime C++ style compiled functions can be used with both the the C++ and Pytho
 .. note::
 
     If you wish to store RTC agent functions in separate files :func:`newRTCFunction()<flamegpu::AgentDescription::newRTCFunction>` can be replaced with :func:`newRTCFunctionFile()<flamegpu::AgentDescription::newRTCFunctionFile>`, instead passing the path to the agent function's source file (relative to the working directory at runtime). This will allow them to be developed in a text editor with C++ syntax highlighting.
+    
+
+To optimise the loading of RTC agent functions, they are only compiled when changes are detected, with compiled agents being cached both in memory during execution and to the a temporary directory on disk between executions. The internal class :class:`JitifyCache<flamegpu::util::detail::JitifyCache>` can be used to configure the in-memory cache by retrieving the singleton via :func:`getInstance()<flamegpu::util::detail::JitifyCache::getInstance>`.
+
+The on-disk cache may grow over time if you are using many different versions of flame gpu and compiling many different agent functions (e.g. running the test suites). If you wish to purge the on-disk cache this can be achieved via the below command.
+
+.. tabs::
+
+  .. code-tab:: cpp C++
+
+    #include "flamegpu/util/detail/JitifyCache.h"
+
+    flamegpu::util::detail::JitifyCache::clearDiskCache()
```

Comment:
We shouldn't be advertising parts of `detail` for users to call, its not part of the "public" API.

Reply:
How else do you want to expose this then. I have documented it as internal in the above text.

Reply:
This is a case of, they shouldn't **need** it, but it's there if they want.

Reply:
Argument for moving `JitifyCache` out of `detail` and just leaving it in `util`?

Reply:
Depends on how much of the rest of JitifyCache (and even the name JitifyCache) wants to be fixed in the public API. 

`loadKernel` doesn't seem like something we should be exposing. 
The various `useX`/`clearX` methods maybe? 

Alternatively we could add a new file / static method somewhere, which just calls `JitifyCache::clearDiskCache`. Could just be a `flmamegpu::util` namespace scoped static method like `cleanup()`?

Reply:
Fair, the configuration options of `JitifyCache` are useful but only really for users who don't want defaults and cant configure at cmake time (Python).

I need to do some primage stuff now, so will probably return to this later today/next week.